CMakeLists.txt

```diff
@@ -260,7 +260,7 @@ if(RDK_BUILD_PYTHON_WRAPPERS)
   if(WIN32 OR "${Py_ENABLE_SHARED}" STREQUAL "1")
     target_link_libraries(rdkit_py_base INTERFACE ${PYTHON_LIBRARIES} )
   endif()
-  
+
```

Comment:
Please ignore changes to this file - I needed to tweak it for my local build, but I'll remove it from the ultimate merge